src/systems/ec/resources/icons/src/ui/fullscreen.svg

```diff
@@ -0,0 +1 @@
+<svg width="16" height="16" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><defs><path d="m14.45045,13.35954l0,-2.60442l1.54955,0l0,5.24488l-5.24488,0l0,-1.54955l2.60442,0l-3.35135,-3.35135l1.09091,-1.09091l3.35135,3.35135zm-3.35135,-7.36445l-1.09091,-1.09419l3.35135,-3.35135l-2.60442,0l0,-1.54955l5.24488,0l0,5.24488l-1.54955,0l0,-2.60442l-3.35135,3.35463zm-6.1982,4.0131l1.09419,1.09091l-3.35463,3.35135l2.60442,0l0,1.54955l-5.24488,0l0,-5.24488l1.54955,0l0,2.60442l3.35135,-3.35135zm0.34398,-8.45864l-2.60442,0l3.35463,3.35135l-1.09419,1.09419l-3.35135,-3.35463l0,2.60442l-1.54955,0l0,-5.24488l5.24488,0l0,1.54955z" id="a"/></defs><use xlink:href="#a"/></svg>
```

Comment:
All our current icons use a `width="24" height="24"` base. Is it possible to have the same base for this one too? (question directed to the designers)